packages/office-ui-fabric-react/src/components/Calendar/Calendar.scss

```diff
@@ -531,6 +583,26 @@ $Calendar-dayPicker-margin: 10px;
     .monthPicker {
       display: block;
     }
+
+  }
+
+  // Do not hide Month picker if it is the only calendar
+  .pickerOnlyMonthPickerVisible, .overlayedCalendars{
```

Comment:
This seems a bit confusing. It seems like in order to have overlayed calendars, it is a requirement that we are showing both the day picker and the month picker, right? So we would never be in a situation where we would be only showing the month picker and also overlayed calendars. Am I missing something?